pkg/providers/instance/instance.go

```diff
@@ -189,6 +186,14 @@ func (p *Provider) Delete(ctx context.Context, id string) error {
 	return nil
 }
 
+func (p *Provider) CreateTags(ctx context.Context, id string, tags []*ec2.Tag) error {
```

Comment:
```suggestion
func (p *Provider) CreateTags(ctx context.Context, id string, tags map[string]string) error {
```

Consider taking in a map here